auth.c

```diff
@@ -934,6 +939,10 @@ subprocess(const char *tag, struct passwd *pw, const char *command,
 		restore_uid();
 		return 0;
 	}
+	/* If the user's SID is the System SID and sshd is running as
```

Comment:
Can you please move this and line 902 to line 971.
Just want to have the code near where it's used and also this whole thing is windows specifc.

Reply:
Absolutely. 
I took "whole thing is windows specifc" to mean that the use of IsWellKnownSid didn't need to be in a "#ifdef WINDOWS", so I collapsed it all into the if statement. If I read that incorrectly, please let me know as I'm also happy to had the indef and variable to that specific section.

Reply:
Thank you for fixing it. 
Could you please fix the comments in your other PR (https://github.com/PowerShell/openssh-portable/pull/478).
I would like to verify the changes before merging. Please provide the steps to verify.